sycl-fusion/passes/kernel-fusion/SYCLKernelFusion.cpp

```diff
@@ -34,8 +37,62 @@ constexpr static StringLiteral KernelArgType{"kernel_arg_type"};
 constexpr static StringLiteral KernelArgBaseType{"kernel_arg_base_type"};
 constexpr static StringLiteral KernelArgTypeQual{"kernel_arg_type_qual"};
 
+constexpr StringLiteral SYCLKernelFusion::NDRangeMDKey;
+constexpr StringLiteral SYCLKernelFusion::NDRangesMDKey;
+
+struct FusedKernel {
```

Comment:
I don't think that `FusedKernel` and `FusedKernelFunction` are good names for this and the following struct, because they do not represent the fused kernel (i.e the outcome of fusion), but rather the input kernels to the fusion process.

Reply:
Renamed to `InputKernel` and `InputKernelFunction`